debuggers/gdb/shim.js

```diff
@@ -232,9 +232,9 @@ function Executable() {
     this.spawn = function(callback) {
         var args = ["--once", ":"+GDB_PORT, BIN].concat(ARGS);
         this.proc = spawn("gdbserver", args, {
-            detached: false,
+            detached: true,
```

Comment:
Signals to the shim get distributed to all its child processes, unless they're detached into their own group. If the gdbserver was not detached, we lose control over signals. For example, create a C program with an infinite loop, like this:
```c
int main(void) {
    int i = 0;
    while (1) { i = 1; }
}
```
Then debug, and when it's running hit ctrl-c. With `detached: true`, we can pause execution (as is expected with normal debugging in GDB; a second ctrl-c will stop debugging). With `detached: false`, SIGINT gets sent to gdbserver and the entire debug process is destroyed with only one ctrl-c.